snuba/cli/subscriptions.py

```diff
@@ -146,6 +158,8 @@ def subscriptions(
                     consumer_group,
                     auto_offset_reset=auto_offset_reset,
                     bootstrap_servers=bootstrap_servers,
+                    queued_max_messages_kbytes=queued_max_messages_kbytes,
+                    queued_min_messages=queued_min_messages,
```

Comment:
I wonder what happens to librdkafka if we create two kafka consumers with the confluent library (this one and the commit log one) to the same broker but with different settings. 
Did you by chance try ?